README.md

```diff
@@ -0,0 +1,69 @@
+# ComLaude/php-formatter
+Simple PhpAmqpLib wrapper for interaction with RabbitMQ 
```

Comment:
Here needs to be updated

Reply:
this is funny you comment was outdated even though it's posted AFTER my commit :D